scripts/analyze_provider_raw.py

```
@@ -0,0 +1,89 @@
+        
+        # 1. Fetch all series to get IDs
+        print("Fetching series list...")
+        all_series = client.get_series()
+        print(f"Found {len(all_series)} series.")
+        
+        # 2. Pick top 5 series to analyze in depth (or scan all if user wants, but that's slow)
+        # We'll scan a few to demonstrate
+        
+        duplicates_found = 0
+        
+        for i, series_info in enumerate(all_series[:10]):
```

Ruff (0.14.7)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)